rasa/utils/tensorflow/tf_model_data.py

```diff
@@ -185,61 +220,23 @@ def balance(self, batch_size: int, shuffle: bool) -> None:
             for v in values:
                 final_data[k].append(np.concatenate(np.array(v)))
 
-        self.data = final_data
-
-    def get_number_of_examples(self) -> int:
-        """Obtain number of examples in session data.
-
-        Raise a ValueError if number of examples differ for different data in
-        session data.
-        """
-
-        example_lengths = [v.shape[0] for values in self.data.values() for v in values]
-
-        # check if number of examples is the same for all values
-        if not all(length == example_lengths[0] for length in example_lengths):
-            raise ValueError(
-                f"Number of examples differs for keys '{self.data.keys()}'. Number of "
-                f"examples should be the same for all data."
-            )
-
-        return example_lengths[0]
-
-    def get_feature_dimension(self, key: Text) -> int:
-        """Get the feature dimension of the given key."""
-
-        number_of_features = 0
-        for data in self.data[key]:
-            if data.size > 0:
-                number_of_features += data[0].shape[-1]
-
-        return number_of_features
-
-    def convert_to_tf_dataset(
```

Comment:
was this a duplicate?

Reply:
yes, it did the same thing as `as_tf_dataset` and was never called